pkg/make_rpm.py

```diff
@@ -155,11 +155,11 @@ class RpmBuilder(object):
   TEMP_DIR = 'TMP'
   DIRS = [SOURCE_DIR, BUILD_DIR, TEMP_DIR]
 
-  def __init__(self, name, version, release, arch, rpmbuild_path, debug=False):
+  def __init__(self, name, version, release, target_platform, rpmbuild_path, debug=False):
```

Comment:
Please add this as a new option in addition to arch. Both have their place.  
It is actually a bug that arch is not being used, it should be. When we are missing platform, we can default arch from the target CPU for the toolchain.